src/runtime_src/xdp/profile/plugin/aie_profile/aie_plugin.cpp

```diff
@@ -146,6 +145,36 @@ namespace xdp {
       {"dma_stalls_mm2s",       {XAIE_EVENT_DMA_MM2S_0_STALLED_LOCK_ACQUIRE_MEM,
                                  XAIE_EVENT_DMA_MM2S_1_STALLED_LOCK_ACQUIRE_MEM}}
     };
+
+    // String event values for guidance and output
+    mCoreEventStrings = {
```

Comment:
It would be better if we instead provided a function that converts an XAie_Events enum to a string. That way, it would automatically pick up any adjustments to metric sets.

Reply:
good idea. but it'd need to contain all enums which is a large amount. We can add it in future.

Reply:
filed a CR for that